metricbeat/module/nats/connections/_meta/fields.yml

```diff
@@ -0,0 +1,17 @@
+- name: connections
+  type: group
+  description: >
+    Contains nats connection related metrics
+  fields:
+    - name: server_id
+      type: string
+      description: >
+        The server ID
+    - name: now
```

Comment:
Same for this field, should be one level up. In general I wonder if we need it as I would hope the time is the same as the collection timestamp?

Reply:
I agree with moving the field one level up to ΜoduleFields.
As far as the need of this field, I agree that it will be the same as the collection timestamp normally, but it might be useful if one collects data from remote nats servers and want to detect time diffs between the clocks of the different machines.
Also it's information comes over http and again if you have to gather data from remote servers the transfer may add time diff.
I have met similar scenarios in the past. 

No strong opinion just discussing :-)

Reply:
SGTM. Let's keep it.